content/en/docs/ibm/deploy/authentication.md

```diff
@@ -85,13 +136,24 @@ These instructions are using Kubeflow dashboard as an example, however, they app
         privateKey: /etc/istio/ingressgateway-certs/tls.key
         serverCertificate: /etc/istio/ingressgateway-certs/tls.crt
     ```
-7. Verify that the traffic is routed via HTTPS by using the value of above `Hostname` in your browser. It should redirect traffic from an HTTP address to HTTPS address automatically.
 
-**Note**: The certificates for the NLB DNS host secret expire every **90** days. The secret in the `istio-system` namespace is automatically renewed by IBM Cloud Kubernetes Service 37 days before it expires. After this secret is updated, you must manually copy it to the `istio-ingressgateway-certs` secret by repeating commands in step 6.
+7. Verify that the traffic is routed via HTTPS by using the value of 
```

Comment:
this should be step 8.

Reply:
Thanks for spotting this. Can you fix it in your repo please so it displays "8. Verify..."?

> it's just those line feeds make code review hard. do you use a fixed width to format markdown?

Wrapping at (or around) 80 characters per line is a common practice (we can treat this as docs as code). We should ideally implement this in a doc test, as some lines can become infinitely long and, therefore, also hard to review.

Reply:
> Wrapping at (or around) 80 characters per line is a common practice (we can treat this as docs as code). We should ideally implement this in a doc test, as some lines can become infinitely long and, therefore, also hard to review.

It totally makes sense when we identify this out of this PR. it's just inconsistent with the rest of Markdown in this repo. I would suggest making this rule explicit and possibly enforce via a future linter, then I'm happy to accept it, or I would have to cherry-pick your commit later.